dts/msm8226/msm8926-samsung-r04.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+        qcom,msm-id =  <0x6E08FF01 4 0x20000>;
+	kmini3g {
+		model = "[ATLANTIC-3G] Rev-03-MTP-v2";
```

Comment:
if this device is "atlantic3g" call the node and compatible this way.
Take the msm-id from the same file you took this name from.

please keep this "model" property containing the "marketing" name of the device though (what's written on the box)

Reply:
i updated the model and called node and compatible